### DISCUSSION TITLE
Easy way to output ```console.print(table)``` to an email

### DISCUSSION THREAD

**Discussion by rjsears** [Textualize/rich#1556] [Q&A] [answered]
Not sure if this is something that fits here, but I was wondering is there is an easy way  to output tables for sending via emails as opposed to the console. I have built some nice reports utilizing tables and I would also like to get these reports emailed to me, but I am just getting a ton os ansi color codes and nothing like what outputs to the screen. 

Maybe someone has some pointers.

Thanks!!

**Comment by willmcgugan** [comment, marked as answer]
See the docs on [exporting](https://rich.readthedocs.io/en/latest/console.html#exporting). You can use the `export_text` method which will return a string without style codes.

**Reply by rjsears** [reply]
Thank you very much for the quick reply, that is exactly what I was looking for, just didn't catch it!
